### DISCUSSION TITLE
what particular problems can be resolved by EOM?

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#59] [Mixtures and flexible systems]
On 2007-12-05 23:33:18, user polymer wrote:

Hi,

I am new to ATSAS. After reading the manual of EOM, I am not clear what particular problems can be resolved by EOM. In which case I should use EOM.  Thanks.

**Comment by franke-biosaxs** [comment]
On 2007-12-06 17:58:24, user Saxologos wrote:

EOM is a method designed for proteins with intrinsic flexibility.
Either natively unfolded proteins or proteins with flexible linkers. It tries to fit several randomly generated models to the same experimental SAXS data.
It can assess the different conformations of a flexible protein in solution i.e. which part of the potential space it can explore.
In short, you should try to use EOM when you have proteins with high flexibility.
